src/chocolatey/infrastructure.app/commands/ChocolateySourceCommand.cs

```diff
@@ -28,7 +28,7 @@ namespace chocolatey.infrastructure.app.commands
 
     [CommandFor(CommandNameType.sources)]
     [CommandFor(CommandNameType.source)]
-    public sealed class ChocolateySourceCommand : ICommand
+    public sealed class ChocolateySourceCommand : IListCommand<ChocolateySource>

```

Comment:
This extends ICommand by adding a *list* function without forcing every command to implement list.  I could make it an additional interface instead of an inherited one, but this seemed natural to me. Is there some objection?